.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: 🐞 Bug report
+about: Tell us about a bug you found
+title: ''
+labels: '🐞 Bug, ⛑ Needs triage'
+assignees: ''
+
+---
+
+### Precheck
+
+- [ ] Do a quick search and make sure the bug has not been reported yet.
+- [ ] Please disclose security vulnerabilities privately at [oss@kommit.co](mailto:oss@kommit.co).
+- [ ] Finally, be nice and have fun!.
+
+### Description
+
+A clear and concise description of what the bug is.
+
+### Environment
+
+* Operating system:
```

Comment:
Include other details you may need, for example, rust version, soroban version, etc...